gnovm/pkg/gnolang/gno_test.go

```diff
@@ -455,6 +455,10 @@ func BenchmarkBenchdata(b *testing.B) {
 				name += "_param:" + param
 			}
 			b.Run(name, func(b *testing.B) {
+				if strings.HasPrefix(name, "matrix.gno_param") {
+					b.Skip("it panics causing an error when parsing benchmark results")
```

Comment:
Can you please elaborate a bit on this in a comment before the skip? 👀

Reply:
Added acomment